src/hotspot/os/windows/os_windows.cpp

```diff
@@ -3265,8 +3285,22 @@ void os::large_page_init() {
   _large_page_size = large_page_init_decide_size();
   const size_t default_page_size = os::vm_page_size();
   if (_large_page_size > default_page_size) {
-    _page_sizes.add(_large_page_size);
+ #if !defined(IA32)
+      // Check if the OS version is 11 or higher
```

Comment:
Again, we need to update this comment or remove it altogether.

Reply:
done